TypeCobol/Compiler/Directives/TypeCobolOptions.cs

```diff
@@ -44,12 +44,18 @@ public bool UseEuroInformationLegacyReplacingSyntax
         private bool _useEuroInformationLegacyReplacingSyntax;
 #endif
 
+        /// <summary>
+        /// Shall we use Check if a End statement is aligned with the matching opening statement.
+        /// </summary>
+        public bool CheckEndAlignment { get; set; } = true;
```

Comment:
The default value should be false (see further comments) but i guess it is easier to make it true for the testing. This needs to be discussed.

Reply:
True is the default value